<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleThresholdDialog</class>
 <widget class="QDialog" name="DoubleThresholdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二值化：双阈值调节</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
</string>
  </property>
  <widget class="QSlider" name="leftThresholdSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rightThresholdSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="leftThreshold">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>53</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>左阈值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightThreshold">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>113</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>右阈值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftThresholdValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>53</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightThresholdValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>113</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conformButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
